Board.java

```diff
@@ -1,97 +1,46 @@
-         return 44;
-      }
-      if(location == 47){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 26);
-         return 26;
-      }
-      if(location == 49){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 11);
-         return 11;
-      }
-      if(location == 51){
-         System.out.println("Congratulations, you landed on a ladder taking you from " + location + " to " + 67);
-         return 67;
-      }
-      if(location == 56){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 53);
-         return 53;
-      }
-      if(location == 62){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 19);
-         return 19;
-      }
-      if(location == 64){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 60);
-         return 60;
-      }
-      if(location == 71){
-         System.out.println("Congratulations, you landed on a ladder taking you from " + location + " to " + 91);
-         return 91;
-      }
-      if(location == 80){
-         System.out.println("Congratulations, you landed on a ladder taking you from " + location + " to " + 100);
-         return 100;
-      }
-      if(location == 87){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 24);
-         return 24;
-      }
-      if(location == 93){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 73);
-         return 73;
-      }
-      if(location == 95){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 75);
-         return 73;
-      }
-      if(location == 98){
-         System.out.println("Uh oh! You landed on a chute taking you from " + location + " to " + 78);
-         return 73;
-      }
-      else{
-         int newLocation = location + spin;
-         while(newLocation > 100){
-            newLocation --;
-         }
-         System.out.println("You advance from " + location + " to " + newLocation);
-         return newLocation;
-      }
+   private static final int[][] LADDERS = {
+     {1, 38}, {4, 14}, {9, 31}, {21, 42}, {28, 84}, {36, 44}, {51, 67}, {71, 91}, {80, 100}};
+   private static final int[][] CHUTES = {
+     {16, 6}, {47, 26}, {49, 11}, {56, 53}, {62, 19}, {64, 60}, {87, 24}, {93, 73}, {95, 73}, {98, 73}};
+   private static String CONGRATS = "Congratulations, you landed on a ladder taking you from ";
+   private static String UH_OH = "Uh oh! You landed on a chute taking you from ";
+   private static String TO = " to ";
+
+   private static int adjustedNewPositionWith(
+      int[][] shortcutsData, int position)
+   {
+      int newPosition = 0;

```

Comment:
i don't think u know how to use `null` and `Integer` instead of plain `int` so had 2 resort 2 using this stupid `substitute` value and code a logic around it.